x-pack/plugins/uptime/public/components/common/charts/duration_chart.tsx

```diff
@@ -113,7 +113,7 @@ export const DurationChartComponent = ({
             tickFormat={timeFormatter(getChartDateLabel(min, max))}
           />
           <Axis
-            domain={{ min: 0 }}
+            domain={{ min: 0, fit: false }}
```

Comment:
what impact this change will have?

Reply:
Short answer: Nothing. We deprecated `yScaleToDataExtent` in favor of this a while ago and finally fully removed the deprecated api. The function is exactly the same.